jp.go

```diff
@@ -116,11 +137,8 @@ func runMain(c *cli.Context) int {
 		os.Stdout.WriteString(converted)
 	} else {
 		var toJSON []byte
-		if c.Bool("compact") {
-			toJSON, err = json.Marshal(result)
-		} else {
-			toJSON, err = json.MarshalIndent(result, "", "  ")
-		}
+		var jsonWriter = getJSONWriter(c.Bool("compact"))
+		toJSON, err = jsonWriter(result)
```

Comment:
Something like this looks simpler to me:
```suggestion
		if c.Bool("compact") {
			toJSON, err = jsoncolor.Marshal(result)
		} else {
			toJSON, err = jsoncolor.MarshalIndent(result, "", "  ")
		}
```

Reply:
Granted.